src/App.jsx

```diff
@@ -4,14 +4,16 @@
 import './App.css';
 import DonationForm from './components/DonationForm/DonationForm';
 import DropZone from './components/DropZone/DropZone';
-import uploadImage from './utils/furnitureUtils';
+import uploadImage from './utils/FurnitureUtils'; // was originally 'FurnitureUtils' should probably be this
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./utils/FurnitureUtils"